Makefile

```diff
@@ -6,6 +6,8 @@ lint:
 
 test: generate lint
 	go test -race -cover ./...
+	go build ./cmd/tengo
+	./tengo ./testdata/cli/test.tengo
```

Comment:
I think this should be `go run ./cmd/tengo ./testdata/cli/test.tengo`.

Reply:
Done